data/common.yaml

```diff
@@ -0,0 +1,54 @@
+---
+ntp::authprov: ~
```

Comment:
yes. `~` translates to ruby `nil`, which is `undef` in the puppet4 ruby api.

On 3 August 2016 at 15:57, Alessandro Franceschi <notifications@github.com>
wrote:

> In data/common.yaml
> <https://github.com/puppetlabs/puppetlabs-ntp/pull/332#discussion_r73346135>
> :
>
> > @@ -0,0 +1,54 @@
> > +---
> > +ntp::authprov: ~
>
> Has the tilde here some special meaning? (undef?)
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/puppetlabs/puppetlabs-ntp/pull/332/files/64db8cd4fd3d437b79e8d8802d64fdc0420a2a4d#r73346135>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AABsQE681uJmD67iJhNyc8bfVt-gOcqwks5qcKx3gaJpZM4JBM8x>
> .
>